GNU Make 3.81 fails on mac sierra

After a fresh clone I ran make as instructed.
I tried to figure it out what was going wrong but after chasing my tail through the makefiles I gave up.

The error output is a combination of a bunch of warning complaining about '-n' target being given more than once in the same rule, e.g., 

> working/make_final.mk:10: target `-n' given more than once in the same rule.


with the final death blow

> make[1]: *** No rule to make target `l/accented/squashed-accented.lst', needed by `working/with-accents.lst'.  Stop.
> make: *** [scowl] Error 2

I started debugging this a little bit more.

The problems start on this line: https://github.com/en-wl/wordlist/blob/master/scowl/Makefile#L53

Running `/usr/bin/make` on macOS 10.13.2 under scowl/ directory (with my PATH fixed up as described earlier https://github.com/en-wl/wordlist/issues/198#issuecomment-346679997)

```
$ make --version
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
```

```
$ make -d 2>&1 | awk '$1!=p{print;p=$1}' # deduplicate repeated consecutive lines
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
Reading makefiles...
Makefile:53: target `-n' given more than once in the same rule.
Reading makefile `working/split_list.mk' (search path) (don't care) (no ~ expansion)...
working/split_list.mk:1: warning: overriding commands for target `-n'
Makefile:53: warning: ignoring old commands for target `-n'
working/split_list.mk:1: target `-n' given more than once in the same rule.
working/split_list.mk:2: warning: overriding commands for target `-n'
working/split_list.mk:1: warning: ignoring old commands for target `-n'
working/split_list.mk:2: target `-n' given more than once in the same rule.
working/split_list.mk:3: warning: overriding commands for target `-n'
working/split_list.mk:2: warning: ignoring old commands for target `-n'
working/split_list.mk:3: target `-n' given more than once in the same rule.
working/split_list.mk:4: warning: overriding commands for target `-n'
working/split_list.mk:3: warning: ignoring old commands for target `-n'
working/split_list.mk:4: target `-n' given more than once in the same rule.
working/split_list.mk:5: warning: overriding commands for target `-n'
working/split_list.mk:4: warning: ignoring old commands for target `-n'
working/split_list.mk:5: target `-n' given more than once in the same rule.
working/split_list.mk:6: warning: overriding commands for target `-n'
working/split_list.mk:5: warning: ignoring old commands for target `-n'
working/split_list.mk:6: target `-n' given more than once in the same rule.
working/split_list.mk:7: warning: overriding commands for target `-n'
working/split_list.mk:6: warning: ignoring old commands for target `-n'
working/split_list.mk:7: target `-n' given more than once in the same rule.
working/split_list.mk:8: warning: overriding commands for target `-n'
working/split_list.mk:7: warning: ignoring old commands for target `-n'
working/split_list.mk:8: target `-n' given more than once in the same rule.
working/split_list.mk:9: warning: overriding commands for target `-n'
working/split_list.mk:8: warning: ignoring old commands for target `-n'
working/split_list.mk:9: target `-n' given more than once in the same rule.
working/split_list.mk:10: warning: overriding commands for target `-n'
working/split_list.mk:9: warning: ignoring old commands for target `-n'
working/split_list.mk:10: target `-n' given more than once in the same rule.
Reading makefile `working/make_final.mk' (search path) (don't care) (no ~ expansion)...
working/make_final.mk:1: warning: overriding commands for target `-n'
working/split_list.mk:10: warning: ignoring old commands for target `-n'
working/make_final.mk:1: target `-n' given more than once in the same rule.
working/make_final.mk:2: warning: overriding commands for target `-n'
working/make_final.mk:1: warning: ignoring old commands for target `-n'
working/make_final.mk:2: target `-n' given more than once in the same rule.
working/make_final.mk:3: warning: overriding commands for target `-n'
working/make_final.mk:2: warning: ignoring old commands for target `-n'
working/make_final.mk:3: target `-n' given more than once in the same rule.
working/make_final.mk:4: warning: overriding commands for target `-n'
working/make_final.mk:3: warning: ignoring old commands for target `-n'
working/make_final.mk:4: target `-n' given more than once in the same rule.
working/make_final.mk:5: warning: overriding commands for target `-n'
working/make_final.mk:4: warning: ignoring old commands for target `-n'
working/make_final.mk:5: target `-n' given more than once in the same rule.
working/make_final.mk:6: warning: overriding commands for target `-n'
working/make_final.mk:5: warning: ignoring old commands for target `-n'
working/make_final.mk:6: target `-n' given more than once in the same rule.
working/make_final.mk:7: warning: overriding commands for target `-n'
working/make_final.mk:6: warning: ignoring old commands for target `-n'
working/make_final.mk:7: target `-n' given more than once in the same rule.
working/make_final.mk:8: warning: overriding commands for target `-n'
working/make_final.mk:7: warning: ignoring old commands for target `-n'
working/make_final.mk:8: target `-n' given more than once in the same rule.
working/make_final.mk:9: warning: overriding commands for target `-n'
working/make_final.mk:8: warning: ignoring old commands for target `-n'
working/make_final.mk:9: target `-n' given more than once in the same rule.
working/make_final.mk:10: warning: overriding commands for target `-n'
working/make_final.mk:9: warning: ignoring old commands for target `-n'
working/make_final.mk:10: target `-n' given more than once in the same rule.
Reading makefile `.symbolic-deps' (search path) (don't care) (no ~ expansion)...
Updating makefiles....
```

some time later...

```
    Considering target file `working/with-accents.lst'.
     File `working/with-accents.lst' does not exist.
      Considering target file `src/make-accent-lists'.
       Looking for an implicit rule for `src/make-accent-lists'.
       Trying pattern rule with stem `make-accent-lists'.
       Looking for a rule with intermediate file `src/make-accent-lists.o'.
        Avoiding implicit rule recursion.
        Trying pattern rule with stem `make-accent-lists'.
        Looking for a rule with intermediate file `src/make-accent-lists.c'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists'.
         Looking for a rule with intermediate file `src/make-accent-lists.y'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists.y'.
         Looking for a rule with intermediate file `src/make-accent-lists.l'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists.l'.
         Looking for a rule with intermediate file `src/make-accent-lists.w'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists.w'.
         Rejecting impossible implicit prerequisite `src/make-accent-lists.w'.
        Trying pattern rule with stem `make-accent-lists'.
        Looking for a rule with intermediate file `src/make-accent-lists.cc'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists.cc'.
        Looking for a rule with intermediate file `src/make-accent-lists.C'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists.C'.
        Looking for a rule with intermediate file `src/make-accent-lists.cpp'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists.cpp'.
        Looking for a rule with intermediate file `src/make-accent-lists.p'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists'.
         Looking for a rule with intermediate file `src/make-accent-lists.web'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists.web'.
        Looking for a rule with intermediate file `src/make-accent-lists.f'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists'.
         Looking for a rule with intermediate file `src/make-accent-lists.F'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists.F'.
         Looking for a rule with intermediate file `src/make-accent-lists.r'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists'.
          Rejecting impossible implicit prerequisite `src/make-accent-lists.l'.
          Trying pattern rule with stem `make-accent-lists.r'.
        Rejecting impossible implicit prerequisite `src/make-accent-lists.F'.
        Trying pattern rule with stem `make-accent-lists'.
        Looking for a rule with intermediate file `src/make-accent-lists.m'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists'.
         Looking for a rule with intermediate file `src/make-accent-lists.ym'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists.ym'.
         Looking for a rule with intermediate file `src/make-accent-lists.lm'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists.lm'.
        Rejecting impossible implicit prerequisite `src/make-accent-lists.r'.
        Trying pattern rule with stem `make-accent-lists'.
        Looking for a rule with intermediate file `src/make-accent-lists.s'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists'.
         Looking for a rule with intermediate file `src/make-accent-lists.S'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `make-accent-lists.S'.
        Rejecting impossible implicit prerequisite `src/make-accent-lists.S'.
        Trying pattern rule with stem `make-accent-lists'.
        Looking for a rule with intermediate file `src/make-accent-lists.mod'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `make-accent-lists.mod'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.c'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.cc'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.C'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.cpp'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.p'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.f'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.F'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.m'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.r'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.s'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.S'.
       Trying pattern rule with stem `make-accent-lists'.
       Rejecting impossible implicit prerequisite `src/make-accent-lists.mod'.
       Trying pattern rule with stem `make-accent-lists'.
       Looking for a rule with intermediate file `src/make-accent-lists.sh'.
        Avoiding implicit rule recursion.
        Trying pattern rule with stem `make-accent-lists.sh'.
       No implicit rule found for `src/make-accent-lists'.
       Finished prerequisites of target file `src/make-accent-lists'.
      No need to remake target `src/make-accent-lists'.
      Considering target file `src/deaccent'.
         Looking for an implicit rule for `src/deaccent.cc'.
         Trying pattern rule with stem `deaccent.cc'.
         No implicit rule found for `src/deaccent.cc'.
         Finished prerequisites of target file `src/deaccent.cc'.
        No need to remake target `src/deaccent.cc'.
       Finished prerequisites of target file `src/deaccent'.
       Prerequisite `src/deaccent.cc' is older than target `src/deaccent'.
      No need to remake target `src/deaccent'.
      Considering target file `src/find-accented'.
         Looking for an implicit rule for `src/find-accented.cc'.
         Trying pattern rule with stem `find-accented.cc'.
         No implicit rule found for `src/find-accented.cc'.
         Finished prerequisites of target file `src/find-accented.cc'.
        No need to remake target `src/find-accented.cc'.
       Finished prerequisites of target file `src/find-accented'.
       Prerequisite `src/find-accented.cc' is older than target `src/find-accented'.
      No need to remake target `src/find-accented'.
      Considering target file `l/accented/squashed-accented.lst'.
       File `l/accented/squashed-accented.lst' does not exist.
       Looking for an implicit rule for `l/accented/squashed-accented.lst'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Looking for a rule with intermediate file `l/accented/squashed-accented.lst.o'.
        Avoiding implicit rule recursion.
        Trying pattern rule with stem `squashed-accented.lst'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.c'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.y'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst.y'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.l'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst.l'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.w'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst.w'.
         Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.w'.
        Trying pattern rule with stem `squashed-accented.lst'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.cc'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst.cc'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.C'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst.C'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.cpp'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst.cpp'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.p'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.web'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst.web'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.f'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.F'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst.F'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.r'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst'.
          Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.l'.
          Trying pattern rule with stem `squashed-accented.lst.r'.
        Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.F'.
        Trying pattern rule with stem `squashed-accented.lst'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.m'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.ym'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst.ym'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.lm'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst.lm'.
        Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.r'.
        Trying pattern rule with stem `squashed-accented.lst'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.s'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst'.
         Looking for a rule with intermediate file `l/accented/squashed-accented.lst.S'.
          Avoiding implicit rule recursion.
          Trying pattern rule with stem `squashed-accented.lst.S'.
        Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.S'.
        Trying pattern rule with stem `squashed-accented.lst'.
        Looking for a rule with intermediate file `l/accented/squashed-accented.lst.mod'.
         Avoiding implicit rule recursion.
         Trying pattern rule with stem `squashed-accented.lst.mod'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.c'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.cc'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.C'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.cpp'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.p'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.f'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.F'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.m'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.r'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.s'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.S'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Rejecting impossible implicit prerequisite `l/accented/squashed-accented.lst.mod'.
       Trying pattern rule with stem `squashed-accented.lst'.
       Looking for a rule with intermediate file `l/accented/squashed-accented.lst.sh'.
        Avoiding implicit rule recursion.
        Trying pattern rule with stem `squashed-accented.lst.sh'.
       No implicit rule found for `l/accented/squashed-accented.lst'.
       Finished prerequisites of target file `l/accented/squashed-accented.lst'.
      Must remake target `l/accented/squashed-accented.lst'.
make: *** No rule to make target `l/accented/squashed-accented.lst', needed by `working/with-accents.lst'.  Stop.
```

--- a/scowl/src/list-classes
+++ b/scowl/src/list-classes
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 if [ "$1" != -1 ]
 then

--- a/scowl/src/list-combin
+++ b/scowl/src/list-combin
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 spellings="`src/list-spellings`"
 classes="`src/list-classes`"

--- a/scowl/src/list-spellings
+++ b/scowl/src/list-spellings
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 if [ "$1" != -1 ]
 then

--- a/scowl/src/split-words-deps
+++ b/scowl/src/split-words-deps
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 echo -n src/split-words  
 echo -n '' working/variant_1.lst

